CMakeLists.txt

```diff
@@ -156,6 +166,140 @@ use_protobuf()
 
 setup_build_boilerplate()
 
+include(ExternalProject)
+set(RIPPLED_NIH_LIBRARIES)
+######################################################################
+###   NIH dep: lz4                                                 ###
```

Comment:
What does NIH stand for? "Not Invented Here"?

Reply:
yes - it's shorter than "third-party" or "external-dependency".

Reply:
That works! I can safely swap out "National Institutes of Health" which made much less sense. :)